docs-chef-io/content/server/upgrades.md

```diff
@@ -37,13 +37,47 @@ Requires License
 Supported Release
 : Chef Infra Server 13 and later are currently supported Chef Software releases. Earlier releases are no longer supported as of 12/31/2020. For more information about supported Chef Software see the [Supported Versions]({{< relref "/versions#supported-commercial-distributions" >}}) documentation.
 
+## Release Specific Notes
+
 ### Upgrading to 14.x
 
 Chef Infra Server 14 moved from Solr to Elasticsearch as its search index.
 {{% server_upgrade_duration %}}
 
 The Chef Infra Server 14 upgrade does not automatically reindex existing external Elasticsearch installations.
 
+#### Upgrading to 14.8
+
+Chef Infra Server 14.8 upgraded PostgreSQL from 9.6 to 13.3. The 14.8 upgrade process requires downtime for vacuuming, upgrading and reindexing the database. There are extra steps that need to be considered when upgrading from versions earlier than 14.8 . The entire upgrade operation is estimated to take 1 minute per 1000 nodes. This process may take longer depending on your server hardware and the size of the node objects in your Chef Infra Server.
+
+##### Database Preparation
+
+1. Running vacuum full on the PostgreSQL database is recommended if auto vacuuming is not set up. This process will reduce the size of the database by deleting unnecessary data and speeds up migration. The vacuum full operation takes around 1 to 2 minutes per GB of data, depending on the complexity of the data. Additionally, the vacuum full operation will need an amount of free disk space at least as large as the size of your database.
+
+    ```bash
+    /usr/lib/postgresql/9.6/bin/vaccumedb --all --full
+    ```
+
+2. To allow for a full restoration to a previous release in the event of a failure we recommend backing up the PostgreSQL database before upgrading. Performing `chef-server-ctl backup` or `knife-ec-backup` is will backup all data. The backup takes around 4 to 5 minutes per GB of data.
+
+##### Upgrade Steps
+
+1. Follow the Chef Infra Server upgrade instructions below.
+
+##### Post-upgrade Steps
+
+1. Reindex the Chef Infra Server
```

Comment:
we also need to specify that this is a one time operation for upgrading to any version >=14.8 the first time.